_shell.c

```diff
@@ -0,0 +1,30 @@
+#include "shell.h"
+
+/**
+ * main - #include "main.h"
+ *
+ * Return: void
+ */
+int main(void)
+{
+	char input[1024];
+	char* args[64];
+
+	while (1)
+	{
+		shell_prompt();
+
+		if (fgets(input, sizeof(input), stdin) == NULL)
```

Comment:
We are not authorized to use `fgets`, you should use `getline`

- Example
`void _print_line(void)
{
  char *s;
  size_t len = 0;
  int r;

  s = malloc(sizeof(char) * 50);
  if (s == NULL)
    free(s);

  printf("\n$ ");
  r = getline(&s, &len, stdin);

  if (r != -1)
    printf("%s", s);

  free(s);
}`